.gitignore

```diff
@@ -19,3 +19,4 @@ build/
 # coverage output
 /coverage/
 
+.env
```

Comment:
Do you use a `.env` file with a tool? Maybe something we can recommend in dev guide for env variables?

Reply:
I used it for setting my local environment and perhaps it does not make sense to add it for the remote repo's codebase. I think it can be removed for the remote repo although if you think it's good to document for the guidelines I can do it. Lmk!